scripts/build-example.sh

```diff
@@ -16,15 +19,24 @@ do
 
   # compile Vega example
   rm -f examples/compiled/$name.vg.json
+
+  start=`python -c 'from time import time; print int(round(time() * 1000))'`
```

Comment:
Why not just use `time`?

Reply:
Or at least `date +%s`.

Reply:
Both of them do not have millisecond precision.

Reply:
See the comment on the top of the file